locales.json

```diff
@@ -851,6 +851,26 @@
       "currentlyNoEvents": "Güncel bir etkinlik yok."
     }
   },
+  "news": {
+    "de": {
+      "local": "LOKALE NEWS",
+      "readMore": "Weiterlesen",
+      "currentlyNoNews": "Es gibt aktuell keine Nachrichten.",
```

Comment:
Heißt das wir benutzen sowohl den Begriff News, als auch Nachrichten in den deutschen Übersetzungen? Find ich persönlich etwas seltsam.

Reply:
Ja is mir auch schon auffgefallen. Ich hab die nur aus dem PR von Kiitos übernommen. Denke wir sollten komplett `News` verwenden oder?

Reply:
Ich wär auch für Neuigkeiten im deutschen

Reply:
Genauso wie hier: https://github.com/Integreat/integreat-react-native-app/blob/develop/ui-ux/2019_10_NewsStream/TuNewsList4.png

Reply:
As discussed [here](https://integreat.slack.com/archives/C011BK71VSM/p1589448612010300) we are using `Nachrichten` for now and will also keep the term `Neugikeiten` as alternative with separate keys to enable fast switching if necessary.